src/Sulu/Bundle/AdminBundle/Command/DownloadBuildCommand.php

```diff
@@ -11,158 +11,19 @@
 
 namespace Sulu\Bundle\AdminBundle\Command;
 
-use Symfony\Component\Console\Command\Command;
-use Symfony\Component\Console\Input\InputInterface;
-use Symfony\Component\Console\Output\OutputInterface;
-use Symfony\Component\Filesystem\Filesystem;
-use Symfony\Contracts\HttpClient\HttpClientInterface;
-
-class DownloadBuildCommand extends Command
+@\trigger_error(
+    \sprintf(
+        'The "%s" class is deprecated since Sulu 2, use "%s" instead.',
+        DownloadBuildCommand::class,
+        UpdateBuildCommand::class
+    ),
+    \E_USER_DEPRECATED
+);
+
+/**
+ * @deprecated use the "UpdateBuildCommand" class instead
+ */
+class DownloadBuildCommand extends UpdateBuildCommand
```

Comment:
Any reason not to decorate the other service? Would be easier to maintain the old command in the future, because we would not have to make sure that the constructor arguments are passed correctly on this one.

Reply:
Fixed!